devtools/conda-envs/test.yml

```diff
@@ -52,9 +52,8 @@ dependencies:
   - moto
 
   # additional pins
-  - openmm=8.1.2
+  - openmm=8.2.0
   - openmmforcefields>=0.14.2
-  - openff-units=0.2.2
```

Comment:
This should now be at openff-units 0.3 (openfe feedstock should have pinned it).

Reply:
Oh got it; do you recommend we pin here?

Reply:
Probably not, we're pretty good at strictly pinning it in the openfe and gufe feedstocks.